content/en/real_user_monitoring/guide/monitor-hybrid-react-native-applications.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Monitor Hybrid React Native Applications
+kind: guide
+description: Guide for monitoring hybrid React Native applications.
+further_reading:
+- link: '/monitors/create/types/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn about RUM Monitors'
+---
+
+## Overview
+
+If you have a hybrid application built on React Native, you can use Datadog to monitor the same application both from the React Native and native sides.
+
+RUM Events are reported as coming from the same application and the same source in Datadog RUM.
+
+## Limitations
+
+- For **errors, resources, and interactions tracking**, the SDKs can work in two ways:
```

Comment:
```suggestion
- For **errors, resources, and interactions tracking**, the SDKs can work in the following ways:
```

Out of habit, I almost never use exact counts

Reply:
that's a good habit - thanks!